android/app/src/main/java/it/teamdigitale/app/italiaapp/MainApplication.java

```diff
@@ -10,6 +10,7 @@
 import com.facebook.react.ReactApplication;
 import com.lewin.qrcode.QRScanReaderPackage;
 import com.imagepicker.ImagePickerPackage;
+import com.oblador.vectoricons.VectorIconsPackage;
```

Comment:
Why this change?

Reply:
I cleaned up code removing unused modules. This is mostly because I tried several solutions before.